src/extensions/jana/JOmniFactory.h

```diff
@@ -55,7 +55,7 @@ class JOmniFactory : public JMultifactory {
     private:
         friend class JOmniFactory;
 
-        void GetData(const JEvent& event) {
+        void GetCollection(const JEvent& event) {
```

Comment:
Ah, the joys of heavily templated code...

Reply:
True. It used to be that if your code compiled when you included a header, you could be pretty sure there were no unresolved symbols in there...